drivers/ovsdriver.go

```diff
@@ -218,11 +240,43 @@ func (d *OvsDriver) DeleteNetwork(id, nwType, encap string, pktTag, extPktTag in
 	return sw.DeleteNetwork(uint16(pktTag), uint32(extPktTag), gateway, tenant)
 }
 
+//CreateBandwidthshaping will chaneg the format of the bandwidth and return the updated BW
+func CreateBandwidthshaping(bandwidth string) uint64 {
+	var (
+		err      error
+		bw       []string
+		rate     uint64
+		bwUint   uint64
+		bwUint64 uint64
+	)
+
+	regex := regexp.MustCompile("[0-9]+")
+	bw = regex.FindAllString(bandwidth, -1)
+	bwUint, err = strconv.ParseUint(bw[0], 10, 64)
+	if err != nil {
+		log.Errorf("error converting bandwidth string to uint64 %+v", err)
+	}
+	if strings.ContainsAny(bandwidth, "g|G") {
+		rate = 1073741824
```

Comment:
+1 . easier to understand if its defined as a constant or something on those lines